iceoryx_posh/source/popo/ports/client_port_data.cpp

```diff
@@ -27,6 +27,8 @@ cxx::VariantQueueTypes getResponseQueueType(const QueueFullPolicy policy) noexce
                                                           : cxx::VariantQueueTypes::FiFo_MultiProducerSingleConsumer;
 }
 
+constexpr uint64_t ClientPortData::HISTORY_CAPACITY_ZERO;
```

Comment:
I think I did not see this used somewhere. Do you have more info?

Reply:
It is just to avoid a magic number and is defined in the header. It is used a few lines further down but since I also use it now outside of the class in the tests it needs a memory location.